styles/onboardingHelper.js

```diff
@@ -0,0 +1,14 @@
+import Expo from 'expo';
+import { Platform } from 'react-native';
+
+export function step8Top() {
```

Comment:
Style: Given have small these are you could put these directly into the onboarding class. This appears to be the only place where they are used.